### PR TITLE
✨ New feature - [ Project TribooAnalytics ] Implement the support for the TribooAnalytics into amp-analytics

### PR DESCRIPTION
**Pull request by TribooAnalytics**
We have released a new Analytics system and would like to enable native support for AMP pages.
The system detects user interactions on pages and obtains navigation information
The GitHub reference is  https://github.com/TribooAnalytics/amphtml

### CODE REVIEW

File: examples/analytics-vendors.amp.html.rej [hunk 1]
```diff
@@ -0,0 +1,30 @@
+--- examples/analytics-vendors.amp.html
```
Comment (powerivq) [inline examples/analytics-vendors.amp.html.rej:1]: You probably don't want to include this.

**Review by powerivq — CHANGES_REQUESTED** [review]

**Comment by powerivq** [comment]
@TribooAnalytics can you also sign CLA?

**Comment by TribooAnalytics** [comment]
The required changes have been committed.
Some CircleCI "checks" tests are still failing, but they concern other vendors AFAICS.
I don't know why CircleCI "end-to-end" test fails.

**Review by erwinmombay — APPROVED** [review]
LGTM. will leave final approval for @powerivq

File: examples/analytics-vendors.amp.html.rej (hunk 1 above)
Comment (TribooAnalytics) [reply examples/analytics-vendors.amp.html.rej]: fixed

**Comment by erwinmombay** [comment]
@TribooAnalytics could you rebase the code. Latest main should have the fix for the deadlinks, ill try and restart the end to end tests to see if its a flake

**Review by powerivq — APPROVED** [review]

**Comment by TribooAnalytics** [comment]
Hi @erwinmombay
we did the rebase

**Comment by TribooAnalytics** [comment]
Thanks everyone for the support

**Pull request merged**